index.html

```diff
@@ -0,0 +1,234 @@
+
+
+            <div class="printing-data">
+                <div class="right-block">
+                    <div class="printing-title">
+                        <h1>Professional Art <br>Printing Data</h1>
+                    </div>
+                    <div class="post-content">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    </div>
+                    <div class="btn-languages ">
+                        <button type="button" class="function">html</button>
+                        <button type="button" class="function">bootstrap</button>
+                        <button type="button"class="function">Ruby</button>
+                    </div>
+                </div>
+                <div class="btn-allProjects">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+
+
+            <div class="printing-data">
+                <div class="right-block">
+                    <div class="printing-title">
+                        <h1>Professional Art <br>Printing Data</h1>
+                    </div>
+                    <div class="post-content">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    </div>
+                    <div class="btn-languages ">
+                        <button type="button" class="function">html</button>
+                        <button type="button" class="function">bootstrap</button>
+                        <button type="button"class="function">Ruby</button>
+                    </div>
+                </div>
+                <div class="btn-allProjects">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+
+            <div class="printing-data">
+                <div class="right-block">
+                    <div class="printing-title">
+                        <h1>Professional Art <br>Printing Data</h1>
+                    </div>
+                    <div class="post-content">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    </div>
+
+                    <div class="btn-languages ">
+                        <button type="button" class="html2">html</button>
+                        <button type="button" class="bootstrap2">bootstrap</button>
+                        <button type="button"class="Ruby2">Ruby</button>
+                    </div>
+                </div>
+                <div class="btn-allProjects">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+</div>
+
+ <div class="about-beryl">
+     <div class="about-content">
+         <h1 class="about-title">About me</h1>
```

Comment:
- [ ] Kindly consider using `h` tag here other than `h1` for the reasons i mentioned above.